live-examples/js-examples/date-valueof.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">var utcDate1 = new Date(Date.UTC(96, 1, 2, 3, 4, 5));
+var utcDate2 = new Date(Date.UTC(0, 0, 0, 0, 0, 0));
+
+console.log(utcDate1.toUTCString());
+// expected output: Fri, 02 Feb 1996 03:04:05 GMT
+
+console.log(utcDate2.toUTCString());
+// expected output: Sun, 31 Dec 1899 00:00:00 GMT
```

Comment:
This isn't an example of `valueOf`.

Reply:
Whoops. Looks like the same example twice.

Reply:
This still needs to be fixed.